src/web/backend/nextcloud/api.py

```diff
@@ -80,18 +115,25 @@ class UserGroupViewSet(NextCloudMixin,
 
     def post(self, username):
         """ Add user to group """
-        group_name = request.json.get('group_name')
-        if not group_name:
-            return jsonify({'message': 'group_name is required parameter'}), 400
-        if not self.nextcloud.get_group(group_name).is_ok:
-            return jsonify({"message": "group not found"}), 404
-        res = self.nextcloud.add_to_group(username, group_name)
-        return self.nxc_response(res)
+        fqdn = request.json.get('fqdn')
```

Comment:
please rename `fqdn` to `group_fqdn`, it will save some brain cycles in the future.

Reply:
+